src/scripts/main.js

```diff
@@ -274,6 +274,12 @@
             if (Date.now() < 1611878400000) {
                 getSettings(settings => recordGWH2020Golems(settings, xhr));
             }
+        } else if (url.includes("mousehuntgame.com/managers/ajax/events/birthday_factory.php")) {
+            // Triggers on Birthday Items claim, room change click (+others, perhaps).
+            // (GMT): Mar 24 2021 15:00:00 GMT+0000 [Birthday 2021 placeholder end date])
+            if (Date.now() < 1616598000000) {
```

Comment:
Just for the future, if we are using something (for example Date.now()) more than once, we should do it once and store it in a variable. (Code reuse best practices i believe)

Reply:
I'm also not a fan of us hardcoding dates throughout the extenstion. It might be better to have them at the very top of the extenstion, so we can see and change them easily if needed.

Another tip is to not use one time functions. We try to make  a function so it works for years to come, not just this year. (As you probably already know).

Reply:
I 100% agree. I followed what we did for golem submissions and in this case it's to short-circuit any late vending machine purchases that might somehow trigger. It's probably not really necessary but feels safe. And means we won't submit next year's blindly.